INSTALL.md

```diff
@@ -0,0 +1,76 @@
+# KernelCi Frontend Ansible installation process
+
+Documentation of the Kernelci frontend installation process
+
+## KernelCI architecture overview
+
+The architecture of kernelCI is split into three main components:
+* Build system
+* Frontend
+* Backend
+
+For more information on the architecture please visit:  http://wiki.kernelci.org/
+
+## Prerequisites
+
+* Host running unix
+	* Git tools
+	* Ansible >= 2.0: http://docs.ansible.com/ansible/latest/intro_installation.html
+* Target server  
+	* Supported OS: Debian (Jessie)
+	* ssh root access to the server
+	* Python >= 2.7.12
+
+## Installing
+
+### Get the source code
+```
+$ git clone https://github.com/kernelci/kernelci-frontend-config.git
+$ git clone https://github.com/kernelci/kernelci-backend-config.git
+```
+
+### Configure Host
```

Comment:
Might be helpful to briefly describe host and target, and especially that the front-end and back-end will be installed to the same target.